hail/python/hail/fs/local_fs.py

```diff
@@ -68,3 +68,6 @@ def remove(self, path: str):
 
     def rmtree(self, path: str):
         rmtree(path)
+
+    def supports_scheme(self, scheme: str) -> bool:
+        return scheme == ""
```

Comment:
Should support `file`

Reply:
LocalFS uses `os` methods and `open`, which don't support `file://` URLs.

Reply:
LocalFS supports it, tested:

```
import hail as hl
hl.init_local()
hl.import_vcf("file:///Users/johnc/Code/hail/hail/1kg/1kg.vcf.bgz")
```